### PR TITLE
Adding Admin Form for Course Entitlements

### PR DESCRIPTION
**Pull request by staubina**
https://openedx.atlassian.net/browse/LEARNER-6169

Added this admin form to be able to see Course Entitlements in the Discovery admin pages.

### CODE REVIEW

File: course_discovery/apps/course_metadata/admin.py [hunk 1]
```diff
@@ -73,6 +73,17 @@ class CourseAdmin(admin.ModelAdmin):
     readonly_fields = ('uuid',)
     search_fields = ('uuid', 'key', 'title',)
 
+@admin.register(CourseEntitlement)
+class CourseEntitlementAdmin(admin.ModelAdmin):
+        list_display = ['course', 'get_course_number', 'mode']
```
Comment (mikix) [inline course_discovery/apps/course_metadata/admin.py]: Your indentation is a bit much. :)

**Review by mikix — APPROVED** [review]
Looks good. Nice course_number touch. Just a comment about indentation.

File: course_discovery/apps/course_metadata/admin.py (hunk 1 above)
Comment (staubina) [reply course_discovery/apps/course_metadata/admin.py]: 👍

**Review by jlajoie — APPROVED** [review]
Though we shouldn't need to manually modify the course_metadata tables themselves, it is nice to have this to fix upstream errors that get persisted downstream, so I'd say this is a reasonable change.

**Pull request merged**